src/Components/Facility/ConsultationForm.tsx

```diff
@@ -101,11 +137,17 @@ export const ConsultationForm = (props: any) => {
         if (res && res.data) {
           const formData = {
             ...res.data,
-            hasSymptom: !!res.data.symptoms && !!res.data.symptoms.length && !!res.data.symptoms.filter((i: number) => i !== 1).length,
-            otherSymptom: !!res.data.symptoms && !!res.data.symptoms.length && !!res.data.symptoms.filter((i: number) => i === 9).length,
-            admitted: res.data.admitted ? String(res.data.admitted) : 'false',
-            admitted_to: res.data.admitted_to ? res.data.admitted_to : '',
-            category: res.data.category ? res.data.category : '',
+            hasSymptom:
+              !!res.data.symptoms &&
+              !!res.data.symptoms.length &&
+              !!res.data.symptoms.filter((i: number) => i !== 1).length,
+            otherSymptom:
+              !!res.data.symptoms &&
+              !!res.data.symptoms.length &&
+              !!res.data.symptoms.filter((i: number) => i === 9).length,
+            admitted: res.data.admitted ? String(res.data.admitted) : "false",
+            admitted_to: res.data.admitted_to ? res.data.admitted_to : "",
```

Comment:
Codacy found an issue: [Identifier 'admitted_to' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808151)